cosmos/x/evm/plugins/txpool/mempool/mempool_test.go

```diff
@@ -155,6 +180,32 @@ func (mplf *mockPLF) Build(event *sdk.Event) (*coretypes.Log, error) {
 	}, nil
 }
 
+func isQueuedTx(mempool *EthTxPool, tx *coretypes.Transaction) bool {
+	_, queued := mempool.Content()
+
+	for _, list := range queued {
+		for _, ethTx := range list {
+			if tx.Hash() == ethTx.Hash() {
+				return true
+			}
+		}
+	}
+	return false
+}
+
+func isPendingTx(mempool *EthTxPool, tx *coretypes.Transaction) bool {
```

Comment:
make the mempool the receiver imo

Reply:
these are lowkey pretty inefficient but thats okay for now

Reply:
ya i'll optimize just wanted to get it working